test/functional/bip68-sequence.py

```diff
@@ -368,12 +368,12 @@ def test_bip68_not_consensus(self):
         assert_equal(self.nodes[0].getbestblockhash(), block.hash)
 
     def activateCSV(self):
-        # activation should happen at block height 432 (3 periods)
-        min_activation_height = 432
+        # activation should happen at block height 576 (4 periods)
+        min_activation_height = 576
         height = self.nodes[0].getblockcount()
         assert(height < min_activation_height)
         self.nodes[0].generate(min_activation_height-height)
-        assert(get_bip9_status(self.nodes[0], 'csv')['status'] == 'active')
+        assert(get_buried_softforks_status(self.nodes[0], 'csv'))
```

Comment:
assert is a statement, not a function. This should be:

`assert get_buried_softforks_status(self.nodes[0], 'csv')`